### PR TITLE
EL-4213 -Select: disable dropdown close on external click

### PR DESCRIPTION
**Pull request by Sdhilipkumar**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4213

#### Description of Proposed Changes
1.Added new input property (autoCloseDropdown).
2. Dropdown should not close on external click when autoCloseDropdown= false,
3. Dropdown should close on external click when autoCloseDropdown= true.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4213-automatic-closing-dropdown-panel

#### Downstream Build(s)

* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-4213-automatic-closing-dropdown-panel~CI/)

### CODE REVIEW

File: e2e/tests/components/select/standard/select.po.spec.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export class SelectPage {
     checkboxMulti = element(by.id('checkbox1'));
     checkboxDisabled = element(by.id('checkbox2'));
     checkboxAllowNull = element(by.id('checkbox3'));
+    checkboxAutoDropdownClose = element(by.id('checkbox6'));
```
Comment (DeanHaire) [inline e2e/tests/components/select/standard/select.po.spec.ts:15]: Can you move this below `checkbox5`?

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Thanks, just a couple of general comments:

1. There's no reason for this setting to be restricted to multiple select, so let's have it apply in single select as well.
2. You've added a new input to Tag Input, so that should be documented.

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Looks good, just missing test coverage of the new input when multiple=false.

**Review by donna-marijne — APPROVED** [review]
Looks good.

**Review by DeanHaire — APPROVED** [review]

**Pull request merged**